cupy/cuda/memory.pyx

```diff
@@ -486,29 +485,60 @@ cdef class SingleDeviceMemoryPool:
     cpdef Py_ssize_t _round_size(self, Py_ssize_t size):
         """Round up the memory size to fit memory alignment of cudaMalloc."""
         unit = self._allocation_unit_size
-        return (((size + unit - 1) // unit) * unit)
+        return ((size + unit - 1) // unit) * unit
 
     cpdef Py_ssize_t _bin_index_from_size(self, Py_ssize_t size):
```

Comment:
return type should be `int` now

Reply:
Now, this is not the index of bins (_free), so naming is not suitable. But, I can not come up with good naming.